layouts/New/layouts.module.css

```diff
@@ -58,32 +58,25 @@
   }
 }
 
-.homeLayout {
+.centeredLayout {
   @apply flex
     w-full
     items-center
-    justify-center
-    px-4
-    py-14
-    md:px-14
-    lg:px-28;
+    justify-center;
 
   main {
     @apply items-center
-      justify-center
-      gap-8
-      md:flex-row
-      md:gap-14
-      xl:gap-28
-      2xl:gap-32;
+      justify-center;
 
     section {
```

Comment:
The centered layout shouldn't have a section IMO, this should all be part of homeLayout